src/utils/getJwt.test.ts

```diff
@@ -7,75 +7,162 @@ vi.mock('node:crypto', () => {
 });
 
 describe('getJwt', () => {
-  const mockConnectedApp = {
-    clientId: 'test-client-id',
-    secretId: 'test-secret-id',
-    secretValue: 'test-secret-value',
-  };
-
   const mockUsername = 'test-user';
   const mockScopes = new Set(['read', 'write']);
 
   afterEach(() => {
     vi.restoreAllMocks();
   });
 
-  it('should include correct header in the token', async () => {
-    const token = await getJwt({
-      username: mockUsername,
-      connectedApp: mockConnectedApp,
-      scopes: mockScopes,
-    });
+  describe('connected-app', () => {
```

Comment:
Kind of a bad diff here. I added a `describe` block for connected apps and another for uat.